crates/hc_bundle/tests/fixtures/web-app/web-happ.yaml

```diff
@@ -3,5 +3,5 @@ manifest_version: "1"
 name: fixture-web-app
 ui:
   path: "./ui.zip"
-happ_manifest:
+happ:
```

Comment:
Slightly unrelated, this name doesn't make sense. There is some code that can automatically bundle the happ because on this path but it's mean to point to a `.happ` file.